app/_includes/prereqs/certificate.md

```diff
@@ -0,0 +1,92 @@
+Create an SSL certificate
+
+1. Generate a private key
+
+    ```sh
+    openssl genpkey -algorithm RSA -out my-key.pem
+    ```
+2. Generate a certificate signing request
+
+    ```
+    openssl req -new -key my-key.pem -out my-csr.pem
+    ```
+3. Create a self-signed certificate 
+
+    ```
+    openssl x509 -req -in my-csr.pem -signkey my-key.pem -out my-cert.pem -days 365
+    ```
+
+4. Create a UUID using the shell: 
+
+    ```
+    uuidgen
+    ```
```

Comment:
Why do we need this step, if two steps down, you tell them to retrieve an autogenerated UUID via API request?